assets/locales/debugger.properties

```diff
@@ -112,6 +112,14 @@ otherEvents=Other
 # source.
 blackBoxCheckboxTooltip=Toggle black boxing
 
+# LOCALIZATION NOTE (sourceSearch.search.key): Key shortcut to open the search
+# for searching within a the currently opened files in the editor
+sourceSearch.search.key=F
```

Comment:
This existed in the `dtd` file but not the properties file so I create a new one that matched our strings.json.

https://dxr.mozilla.org/mozilla-central/source/devtools/client/locales/en-US/debugger.dtd#114

(referenced from here)
https://dxr.mozilla.org/mozilla-central/source/devtools/client/debugger/debugger.xul#211